gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -778,7 +779,17 @@ def _convert_to_variant_record(self, record, infos, formats):
         # Special case: END info value specifies end of the record, so adjust
         # variant.end and do not include it as part of variant.info.
         if k == END_INFO_KEY:
-          variant.end = v
+          # Some VCF files set the number of END info fields to unknown instead
+          # of one, which parses into a list in PyVCF.
+          if isinstance(v, list):
+            if len(v) == 1:
+              variant.end = v[0]
+            else:
+              raise ValueError(
+                  'END INFO field has more than one item: {}'.format(
+                      self._last_record))
+          else:
+            variant.end = v
```

Comment:
Maybe we should move this to a private method like `_get_variant_end`? That way if we ever decide to change the behavior here or the parser changes we can easily refactor.

Reply:
Good idea. Done.